fixtures/enhanced_resolve/test/fixtures/browser-module/package.json

```diff
@@ -1,4 +1,6 @@
 {
+  "exports": {
+  },
```

Comment:
You have quoted `postcss` to be the reason behind the change. And I agree that perhaps in their situation the browser field still should be used (not because I agree with it personally but because this is what webpack and others do). But `postcss` situation is somewhat different than what you have here in this test case.

I think it would make sense to:
- do the browser-based resolution *after* resolving `exports` for relative import sources
- always use the browser-based resolution for absolute/bare import sources 

I didn't test the latter but I did test the first thing with webpack and I think it behaves like I've described. The in-package files are only resolved through the browser field (aka alias field) *after* a successful match is found in `package.json#exports`